tests/TestCase.php

```diff
@@ -73,7 +73,8 @@ public function mockComponent($config = []) {
      */
     protected function expectRedirect($url)
     {
-        $this->setExpectedException('\yii\base\Exception', $this->prepareUrl($url));
+        $url = $this->prepareUrl($url) ? : '/';
+        $this->setExpectedExceptionRegExp('\yii\base\Exception', '#^' . $url . '$#');
```

Comment:
i changed this to regexp, because `setExpectedException` do no action when message value was `/en` and   expected value was `/`